src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -135,6 +137,20 @@ class BaseOptionsSelector extends Component {
         }
     }
 
+    /**
+     * @param {Array<Object>} allOptions
+     * @returns {Number}
+     */
+    getInitiallyFocusedIndex(allOptions) {
+        const indexOfInitiallyFocusedOption = _.findIndex(allOptions, option => option.keyForList === this.props.initiallyFocusedOptionKey);
```

Comment:
@rushatgabhane  updated according to your suggestion, with the minor variation where the `return defaultIndex;` is the last `return` rather than `return intiallyFocusedOptionIndex;`. Since we might add other logic to calculate the focused index in the future, I feel that the `defaultIndex` should be the fallback `return`. Let me know if you feel strongly that `intiallyFocusedOptionIndex` should be the last return instead.

Thanks

```
         if (indexOfInitiallyFocusedOption >= 0) {
             return indexOfInitiallyFocusedOption;
         }

         return defaultIndex;
```